### ISSUE TITLE
[Support]: Feature request - Custom Tooltip renderer (raw html) does not pick up aria-label from tooltip content

### ISSUE THREAD

**Issue by jakebixby** [ag-grid/ag-charts#6076]
### Add a description

I am currently using the [line series tooltip renderer](https://ag-grid.com/charts/options/series/line/#reference-AgLineSeriesOptions-tooltip-renderer) to create a custom tooltip (which works with no issue), however I noticed when the tooltip is rendered, there is no aria label available for screen readers. After some investigation, I found that the [current behavior in the tooltip is to ignore raw content](https://github.com/ag-grid/ag-charts/blob/release-13.0.0/packages/ag-charts-community/src/chart/tooltip/tooltipContent.ts#L103) (i.e. a string of html) when building the description in the `tooltipContentAriaLabel` function. 

I was wondering if it would be possible to parse the raw content for `aria-label` attributes to extract the label for the screen reader.

Thanks!

### Version

13.0.0

### What language are you using?

JavaScript

### What browser are you using?

Chrome

**Comment by AG-DavidG** [comment]
Hello @jakebixby 

Thank you for reporting this - much appreciated!

Based on your description we've now reproduced this issue as a bug.

We have added this bug to our backlog and we are tracking it with the following reference and description:
AG-16833 - [Charts] Custom tooltip renderer produces no screen reader announcement during keyboard navigation

We try to fix bugs from one release to the next, so this should be fixed in the next release or the one after if it was raised too close to the next release date.

See whether this item will be in the next release by checking the NEXT RELEASE checkbox on the product pipeline page:
https://ag-grid.com/charts/pipeline/

The best way to track this is to sign up for AG Charts new release notifications using [the instructions here](https://ag-grid.zendesk.com/hc/en-us/articles/15387001154205-How-do-I-Sign-Up-For-AG-Charts-Release-Notifications). This way you'll know as soon as a new version is out and you can check whether this specific item was implemented on [the changelog page](https://ag-grid.com/charts/changelog/).

Thanks again for bringing this up with us.

Kind regards,

David
